apps/support/Makefile.inc

```diff
@@ -55,9 +55,16 @@ endif
 endif
 LIBPNG_LIBS ?= $(LIBPNG_LIBS_DEFAULT)
 
-LIBJPEG_LIBS = -ljpeg
+# Workaround brew Cellar path for libpng-config output.
+LIBJPEG_LINKER_PATH ?= $(shell echo $(LIBPNG_LIBS_DEFAULT) | sed -e'/-L.*[/][Cc]ellar[/]libpng/!d;s=\(.*\)/[Cc]ellar/libpng/.*=\1/lib=')
```

Comment:
What's going on here?

Reply:
This was done to the toplevel Makefile in a PR a couple weeks or so ago. Brew installs each package in its own directory and then links to the commands from the install directory. The -config command typically returns the per-package directory. This heuristically recognizes that situation and makes a better guess for the case. It means I can build on the Mac and shouldn't affect anything else.

We should probably have a single Makefile include that contains functionality used everywhere, including the top-level. I was hoping we'd be using bazel before we had to start doing stuff like this.